share/man/man4/lagg.4

```diff
@@ -214,7 +214,7 @@ The following example shows how to create an infiniband failover interface.
 .Ed
 .Pp
 Configure two ethernets for failover with static IP in
-.Xr /etc/rc.conf 5 :
+/etc/rc.conf:
```

Comment:
At least do .Pa here.
But i think there is rc.conf man page....

Reply:
okay, there is a reference to rc.conf(5) in the SEE ALSO section, as it was currently written.. it was incorrectly referencing a /etc/rc.conf 5 that did NOT exist.

Reply:
addressed this and added the .Pa macro to the line, confirmed with mantra utility to be operational.